tests/test_pipelines.py

```diff
@@ -10,7 +10,7 @@
 from .pipelines.object_detection import object_detection_pipeline
 
 
-COWSAY_PARAMS = [{"name": "url", "value": "https://helloacm.com/api/fortune/"}]
+COWSAY_PARAMS = [{}]
```

Comment:
From the test failure, this should be `[]` instead of `[{}]`

Reply:
@knkski seems like tests are still failing due to this, can you revisit?